go/vt/mysqlctl/proto/schema.go

```diff
@@ -189,6 +189,12 @@ func (sd *SchemaDefinition) ToSQLStrings() []string {
 // generates a report on what's different between two SchemaDefinition
 // for now, we skip the VIEW entirely.
 func DiffSchema(leftName string, left *SchemaDefinition, rightName string, right *SchemaDefinition, er concurrency.ErrorRecorder) {
```

Comment:
probably need a unit test case or two for this? I think we already have some in there somewhere.